site/en/install/lang_c.ipynb

```diff
@@ -148,7 +148,7 @@
         "  </tr>\n",
         "  <tr>\n",
         "    <td>Windows GPU only</td>\n",
-        "    <td class=\"devsite-click-to-copy\"><a href=\"https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-windows-x86_64-2.11.0.zip\">https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-windows-x86_64-2.11.0.zip</a></td>\n",
+        "    <td class=\"devsite-click-to-copy\"><a href=\"https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-windows-x86_64-2.10.0.zip\">https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-windows-x86_64-2.10.0.zip</a></td>\n",
```

Comment:
Down on this line:

"FILENAME=libtensorflow-cpu-linux-x86_64-2.11.0.tar.gz\n"

It could benefit frim a reminder to change the filename to the one you chose from this table.